ui/components/app/wallet-overview/hooks/useHandleSendNonEvm.ts

```diff
@@ -77,13 +79,13 @@ export const useHandleSendNonEvm = (caipAssetType?: CaipAssetType) => {
         return caipAssetType;
       }
 
-      if (!nativeAssetType) {
+      if (!nativeCurrency) {
         throw new Error(
           'No CAIP asset type provided, and could not find a fallback native asset for the selected account',
         );
       }
 
-      return nativeAssetType;
+      return nativeCurrency as CaipAssetType;
```

Comment:
Looks like we are missing a `if (!selectedNetwork.isEvm)` somewhere 🤔 We shouldn't need to type-cast here.

Should we also throw if the `selectedNetwork` is not EVM?